### DISCUSSION TITLE
Charging limit is limited to 20 increments. Can't lock it to 85 or 90 or 95. Only 60, 80, 100 and such

### DISCUSSION THREAD

**Discussion by CaptainDoubleYou** [seerge/g-helper#3236] [Q&A]
I'm trying to set charging limit to 90 but I can't the slider jumps back to 80 or 100. Nor sure if this is the default behavior or something off on my end but I would like to get 90 or 95 at the expense of 100 or just 80.

**Comment by seerge** [comment]
@CaptainDoubleYou your device simply does not support 85% or 90%. App absolutely intentionally lets you select only values supported by hardware.

You can still set any value in a range between 60% and 80% (i.e. 75% as well) and 100%.
